SonarQube.TeamBuild.PreProcessor/ArgumentProcessor.cs

```diff
@@ -30,12 +30,15 @@ private static class KeywordIds
             public const string ProjectKey = "1";
             public const string ProjectName = "2";
             public const string ProjectVersion = "3";
-            public const string RunnerPropertiesPath = "4";
-            public const string DynamicSetting = "5";
+            public const string InstallLoaderTargets = "4";
```

Comment:
Go ahead. They are only used internally.
FYI Originally the id was an enum; I forgot to assign them meaningful values when I converted it to a string.